src/Serilog.Sinks.XUnit/Sinks/XUnit/TestOutputSink.cs

```diff
@@ -1,19 +1,34 @@
+using Xunit.Sdk;
```

Comment:
Would you mind moving this along with the others?

Reply:
Move it to inside the namespace?

Reply:
Yeah, for consistency's sake

Reply:
Or the other way; move it to outside the namespace? The code is inconsistent here.

Reply:
I think most are inside, though I may have overlooked some at some point.

Reply:
I think it only was inconsistent after my change. 😬